src/object-store/Cargo.toml

```diff
@@ -18,22 +18,21 @@ futures.workspace = true
 lazy_static.workspace = true
 md5 = "0.7"
 moka = { workspace = true, features = ["future"] }
-opendal = { version = "0.45", features = [
+opendal = { version = "0.46", features = [
     "layers-tracing",
-    "rustls",
     "services-azblob",
     "services-fs",
     "services-gcs",
     "services-http",
     "services-oss",
     "services-s3",
-], default-features = false }
```

Comment:
The default features is trimeed as `"services-memory", "reqwest/rustls"` that is exactly we want.